Makefile

```
@@ -52,12 +52,9 @@ whitespace-check: $(all_sources)
 	./hack/whitespace.sh
 
 vet: $(cmd_sources) $(pkg_sources)
-	go vet ./pkg/... ./cmd/...
-
-goimports-check: $(cmd_sources) $(pkg_sources)
-	go run ./vendor/golang.org/x/tools/cmd/goimports -d ./pkg ./cmd
+	go vet --mod=vendor ./pkg/... ./cmd/...
```

the arg takes 1 dash only

```suggestion
	go vet -mod=vendor ./pkg/... ./cmd/...
```